### PR TITLE
log: add logging tools for commands; lxc-usernsexec: cleanup and bugfixes

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: src/lxc/cmd/lxc_usernsexec.c [hunk 1]
```diff
@@ -106,51 +109,60 @@ static void opentty(const char *tty, int which)
...
 
 /*
- * given a string like "b:0:100000:10", map both uids and gids
- * 0-10 to 100000 to 100010
+ * Given a string like "b:0:100000:10", map both uids and gids 0-10 to 100000
+ * to 100010
  */
 static int parse_map(char *map)
 {
+	long host_id, i, ns_id, range, ret;
```
Comment (stgraber) [inline src/lxc/cmd/lxc_usernsexec.c]: why is ret a long here?

Comment (brauner) [reply src/lxc/cmd/lxc_usernsexec.c]: Accident. Let me fix.

Comment (brauner) [reply src/lxc/cmd/lxc_usernsexec.c]: fixed

**Comment by brauner** [comment]
@2xsec, you can add them too if you want. :) I do wonder though whether both `tools` and `cmd`s couldn't just use the standard `log` macros directly. Probably possible since they are now once again integrated into the main LXC tree.

**Pull request merged**